packages/synapse-react-client/src/components/QueryWrapperErrorBoundary.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useCallback, useMemo } from 'react'
+import { ErrorBanner } from './error/ErrorBanner'
+import { useQueryContext } from './QueryContext/QueryContext'
+import { EntityActionsRequired } from './AccessRequirement/EntityActionsRequired'
+import { SynapseClientError, useSynapseContext } from '../utils'
+import {
+  ErrorBoundary,
+  ErrorBoundaryPropsWithComponent,
+} from 'react-error-boundary'
+
+/**
+ * Error boundary that automatically pulls the error from QueryContext.  If 403, shows entity actions required
+ */
+export function QueryWrapperErrorBoundary({
```

Comment:
Replaced `QueryWrapperErrorBanner`. Since table query data is now managed by react-query, there is no 'central' error state stored in QueryWrapper. Instead, will pass errors to an error boundary. This error boundary component will do essentially the same thing as QueryWrapperErrorBanner.